src/device.rs

```diff
@@ -742,11 +1055,23 @@ impl Device {
                                   vertex_stride,
                                   16);
         gl::vertex_attrib_pointer(VertexAttribute::MatrixIndex as gl::GLuint,
-                                  4,
```

Comment:
Some GPUs pay a per-attribute cycle fetch penalty. Let's make this one attribute of four bytes, with x=matrix index, y=color tex index etc.